models/schema.yml

```diff
@@ -279,3 +281,199 @@ models:
         description: "The number of units in inventory of the product variant"
         tests:
           - not_null
+
+  - name: shopify__refunds
+    description: "A table storing information regarding order refunds"
+    columns:
+      - name: refund_id
+        description: "The unique identifier of the refund"
+        tests:
+          - unique
+          - not_null
+      - name: order_id
+        description: "The unique identifier of the order"
+        tests:
+          - not_null
+          - relationships:
+              to: ref('shopify__orders')
+              field: order_id
+      - name: user_id
+        description: "The unique identifier of the user who created the refund"
+        tests:
+          - not_null
+      - name: total_price
+        description: "The total price of the original order"
+        tests:
+          - not_null
+
+metrics:
+  - name: number_of_customers
+    label: Number of Customers
+    model: ref('shopify__customers')
+    description: "The number of customers"
+
+    type: count_distinct
+    sql: customer_id
+
+    timestamp: created_at_timestamp
+    time_grains: [ day, week, month, year ]
+
+    dimensions:
+      - city
+      - country
+
+  - name: number_of_repeat_customers
+    label: Number of Repeat Customers
+    model: ref('shopify__customers')
+    description: "The number of repeat customers"
+
+    type: count_distinct
```

Comment:
I don't see how this is giving you repeat customers

Reply:
I forgot the filter - fixed

Reply:
I'm a little unsure about this. Would it make more sense to just have the customers by orders_count and the user can filter on the frontend (with cube) orders_count > 1

Reply:
you can have either one but I don't see the filter you said you added?

Reply:
I removed `number_of_repeat_customers` in favour of `number_of_customers` + the `orders_count` dimension where the user can create number_of_repeat_customers themselves using their own filter